descriptors/ModuleDescriptor-template.json

```diff
@@ -416,9 +416,71 @@
           "permissionsRequired": ["patron-action-session-storage.expired-session-patron-ids.collection.get"]
         }
       ]
+    },
+    {
+      "id": "check-in-operation-storage",
```

Comment:
I'm not keen on `operation`, yet I've been trying to to think of an alternative, without much success. What do you think of `check-in-storage`?

```suggestion
      "id": "check-in-operation-storage",
```

Reply:
Fine for me, do you mean to rename URL as well?

Reply:
What I'm thinking is that the interface ID is `check-in-storage`, and the paths are something like `check-in-storage/check-ins`

Reply:
This is assuming that FOLIO is still using plural path names for its endpoints